pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -26,12 +26,12 @@ social:
 = I18n.t(:hoc_overview_subtitle)
 
 .full-resource-block{style: "margin-top: 20px;"}
-  %a{href: "/dance"}
-    %img.full-resource-image{src: "/images/dance_party.gif"}
+  %a{href: "/dance-2019"}
+    %img.full-resource-image{src: "/images/dance_party_2019.gif"}
   .tutorial-info{style: "padding: 30px;"}
     %span
       %h2.resource-title= I18n.t(:hoc_overview_dance_title)
-      = I18n.t(:hoc_overview_dance_subtitle)
+      = I18n.t(:hoc_overview_dance_2019_subtitle)
```

Comment:
Does the link to the teacher guide on this page also need to be updated? Is there new curriculum to go with Dance 2019?

Reply:
Good question.  It's going to stay the same.